tests/instrumentors/logging_instrumentor_tests.rb

```diff
@@ -26,6 +26,33 @@
     ]
   end
 
+  tests("connection logger with query as hash").returns(true) do
```

Comment:
Hey, it appears that this test is returning false (instead of true) in CI and failing. Could you dig in and see if you could get that to passing? Thanks!

Reply:
The logfile name can be random so i used Tempfile object create example logfiles that can be appended to. Should work now …

Reply:
commit: bb90e2c should have the patch